controllers/mysql_clustering_test.go

```diff
@@ -80,13 +80,24 @@ func TestDecideNextOperation(t *testing.T) {
 				Wait: false,
 				Operators: []Operator{
 					&configureReplicationOp{
-						index:       1,
-						primaryHost: hostName(0),
+						Index:       1,
+						PrimaryHost: hostName(0),
 					},
 					&configureReplicationOp{
-						index:       2,
-						primaryHost: hostName(0),
+						Index:       2,
+						PrimaryHost: hostName(0),
 					},
+					&setLabelsOp{},
+				},
+			},
+		},
+		{
+			name:  "LabelsAreWrong",
```

Comment:
This test only covers the case "a read-only instance has a wrong label".
I think there is the case "a writable (it's the current primary) instance has a wrong label".  Isn't the test for this case needed?